turbopack/crates/turbopack-core/src/module_graph/mod.rs

```diff
@@ -1267,19 +1304,21 @@ enum SingleModuleGraphBuilderNode {
         module: ResolvedVc<Box<dyn Module>>,
         layer: Option<ReadRef<RcStr>>,
         ident: ReadRef<RcStr>,
+        export: ExportUsage,
     },
     /// A reference to a module that is already listed in visited_modules
     VisitedModule {
         module: ResolvedVc<Box<dyn Module>>,
         idx: GraphNodeIndex,
+        export: ExportUsage,
     },
```

Comment:
I think this is wrong. This would add the module multiple times with different `ExportUsage`s

Reply:
Yep. I had the same conclusion even before reading this comment. Nodes should not have `export`, only edges should have that. It would duplicate nodes per export.